internal/deviations/deviations.go

```diff
@@ -625,3 +625,9 @@ func ISISLspMetadataLeafsUnsupported(dut *ondatra.DUTDevice) bool {
 func QOSQueueRequiresID(dut *ondatra.DUTDevice) bool {
 	return lookupDUTDeviations(dut).GetQosQueueRequiresId()
 }
+
+// EcnMaxMimumThreshold returns the maximum threshold value while comparing for pass or fail conditions.
+// Maximum and mimium threshold value not be same. Anything less Maximum threshold should be 6144 greate then mimnimum threshold value.
+func EcnSameMaxMimumThresholdUnsupported(dut *ondatra.DUTDevice) bool {
```

Comment:
Should we name this function "EcnSameMinMaxThresholdUnsupproted"? and please also update the comment.

Reply:
okay

Reply:
The deviation function name should be the Camel case of the protobuf field name. For example, the func name of field "ecn_same_min_max_threshold_unsupported" is "EcnSameMinMaxThresholdUnsupported".

Reply:
modified